conans/client/detect.py

```diff
@@ -189,7 +189,7 @@ def detect_defaults_settings(output):
     """ try to deduce current machine values without any
     constraints at all
     """
-    output.writeln("\nIt seems to be the first time you run conan", Color.BRIGHT_YELLOW)
+    output.writeln("\nIt seems to be the first time you've ran conan", Color.BRIGHT_YELLOW)
```

Comment:
Interesting.  I am a native speaker and unfortunately I don't know much about the technicalities of grammar - I'm just going off of what feels natural.  I think either of them would work.  In case 1 (you've ran) - in a sense the user has ran the command: they typed it in and hit enter.  I think the past tense makes sense here, because if you asked them "did you run the command?" they would say "yes, I ran it".  Perhaps its more accurate to say they started the command?  In case 2, you could say "you are running" .. in this case, it's more like the computer is talking to the user in real-time.  I think either would be fine, I'm pretty sure that "you run" is not correct in any sense.